editor/editor_toaster.cpp

```diff
@@ -438,6 +439,13 @@ void EditorToaster::close(Control *p_control) {
 	toasts[p_control].popped = false;
 }
 
+void EditorToaster::_close_button_theme_changed(Control *p_close_button) {
+	Button *close_button = (Button *)p_close_button;
```

Comment:
Shouldn't we use `Object::cast_to<Button>`?

Reply:
Done